tests/testthat/test-trim.R

```diff
@@ -16,7 +16,7 @@ test_that("trim works", {
       "test
     "))
   expect_identical("test",
-    trim("      
+    trim("
```

Comment:
@jimhester should I worry about this diff? Elsewhere in this file, I see you've taken measures to prevent RStudio from removing trailing whitespace on save. Shall I assume this whitespace was not load-bearing, i.e. a / the point of the test?

Reply:
probably true, though I unfortunately can't remember for this specific test.

Maybe a better solution to what I did with the tabs is to use "\x20", the hex escape for an ASCII space...

Reply:
Oh right, actually Alan Dipert was who did the mixed tabs and spaces thing